label_studio_converter/converter.py

```diff
@@ -121,6 +122,14 @@ class Converter(object):
             'link': 'https://labelstud.io/guide/export.html#YOLO',
             'tags': ['image segmentation', 'object detection'],
         },
+        Format.YOLO_OBB: {
+            'title': 'YOLOv8-OBB',
+            'description': 'Popular TXT format is created for each image file. Each txt file contains annotations for '
+            'the corresponding image fileThe YOLO OBB format designates bounding boxes by their four corner points '
```

Comment:
replace the description to this:

'description': 'Popular TXT format is created for each image file. Each txt file contains annotations for '
'the corresponding image file. The YOLO OBB format designates bounding boxes by their four corner points '
'with coordinates normalized between 0 and 1, so it is possible to export rotated objects.',